packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -15,6 +15,7 @@ import { withTooltip } from "@core/hocs";
 import dialogClasses, { HvDialogClasses } from "./dialogClasses";
 import { useTheme } from "@core/hooks";
 import { ClassNames } from "@emotion/react";
+import { HvDialogBackdrop } from "./Backdrop";
```

Comment:
I think this component doesn't "deserve" it own file structure and to be exported - we only need it internally for `Dialog.tsx`.

We should have it declared there.
But we definitely don't want this exported to the "public API"

Reply:
I understand. I'll change it again 🤔

Reply:
Updated ⬇️